src/patternfly/components/InputGroup/input-group.scss

```diff
@@ -84,6 +84,10 @@
   textarea {
     min-height: var(--pf-c-input-group__textarea--MinHeight);
   }
+
+  .pf-c-select {
```

Comment:
My apologies, I told you to do the wrong thing in the original issue. If we make the width `auto`, then the toggle button will change size depending on the width of the text in the selected item. The toggle width should remain a fixed width, regardless the selected item's width. So instead, can you remove this CSS and apply a `style="width: [some value];` inline style to the select in the example?

Reply:
I added to the scss file because I couldn't figure out how to do the inline styling in InputGroup.md.  I'll get rid of my addition to the scss and keep trying.

Reply:
Thanks! I left a comment here with a suggestion - https://github.com/patternfly/patternfly/pull/2934/files#r405850893